src/hotspot/share/classfile/verifier.cpp

```diff
@@ -3149,7 +3149,7 @@ void ClassVerifier::verify_return_value(
   if (return_type == VerificationType::bogus_type()) {
     verify_error(ErrorContext::bad_type(bci,
         current_frame->stack_top_ctx(), TypeOrigin::signature(return_type)),
-        "Method expects a return value");
+        "Method descriptor has a void return type");
```

Comment:
I would prefer "Method does not expect a return value".

Reply:
I changed the messages to what you suggested.
Thanks for reviewing this change.
Harold